jobs/build-webapp.groovy

```diff
@@ -642,6 +654,7 @@ def deployAndReport() {
       }
       parallel(jobs);
 
+      createCloudRunTags();
```

Comment:
You could put in a TODO that it's possible to do these in parallel.  Or even just do them in parallel right now!

Reply:
yeah that makes sense! will do.